### DISCUSSION TITLE
Welcome to docsy Discussions!

### DISCUSSION THREAD

**Discussion by emckean** [google/docsy#468] [CSS and other site hacks]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions about Docsy! (For questions about [Hugo](https://gohugo.io/documentation/), please visit the [Hugo Discourse](https://discourse.gohugo.io/))
  * Share ideas.
  * Engage with other community members.
  * Welcome others and be kind! Remember that this is a community we build together 💪.
  * Please remember that participation in this community is governed by Google's [Open Source Community Guidelines](https://opensource.google/conduct/)

**Comment by Symbolics** [comment]
Nice job, it makes good sense to have discussions co-located with github.

How did you manage to put a banner/hero image on the top of the discussion page? Neat trick; I've not seen that before.

**Reply by emckean** [reply]
Thank you! And the image came out of the box. :)
